strings/russian_replacement.json

```diff
@@ -1,6 +1,92 @@
 {
     "full": {
-
+        "hello": "privet",
+		"goodbye": "poka",
+		"yes": "da",
+		"no": "net",
+		"please": "pozhaluysta",
+		"thank you": "spasibo",
+		"thanks": "spasibo",
+		"good": "khorosho",
+		"bad": "ploho",
+		"enemy": "vrag",
+		"love": "lyubov",
+		"hate": "nenavist",
+		"family": "semya",
+		"mother": "mat'",
+		"father": "otets",
+		"dad": "papa",
+		"mom": "mama",
+		"mum": "mama",
+		"brother": "brat",
+		"sister": "sestra",
+		"bitch": "cyка",
+		"vagina": "pizda",
+		"cunt": "pizda",
+		"penis": "khuy",
+		"cock": "khuy",
+		"dickhead": "khuylo",
+		"fucking": "blyat",
+		"fuck": "ebat",
+		"whore": "blyad",
+		"asshole": "muдaк",
+		"shit": "pizdets",
+		"jerK": "muдaк",
+		"king": "korol'",
+		"queen": "koroleva",
+		"prince": "prints",
+		"princess": "printsessa",
+		"duke": "gerzog",
+		"duchess": "gerzoginya",
+		"knight": "druzhina",
+		"soldier": "soldat",
+		"kingdom": "korolevstvo",
+		"dukedom": "gerzogstvo",
+		"grenzelhoft": "grenzeliya",
+		"otava": "otavaiya",
+		"thank": "spasibo",
+		"dame": "dama",
+		"orc": "ork",
+		"homeland": "rodina",
+		"motherland": "rodina",
+		"fatherland": "rodina",
+		"water": "voda",
+		"wine": "vino",
+		"grandmother": "babushka",
+		"grandfather": "dedushka",
+		"grandma": "babushka",
+		"grandpa": "dedushka",
+		"comrade": "tovarishch",
+		"companion": "tovarishch",
+		"friend": "tovarishch",
+		"god": "bog",
+		"goddess": "boginya",
+		"gods": "bogi",
+		"priest": "patriarkh",
+		"priestess": "matriarkh",
+		"church": "tserkov'",
+		"sword": "mech",
+		"shield": "shchit",
+		"axe": "topor",
+		"huzzah": "ura",
+		"huzza": "ura",
+		"yippee": "ura",
+		"hooray": "ura",
+		"hurrah": "ura",
+		"rebellion": "revolyutsiya",
+		"revolution": "revolyutsiya",
+		"captain": "kapitan",
+		"sergeant": "serzhant",
+		"merchant": "torgovets",
+		"trader": "torgovets",
+		"knave": "lzhets'",
+		"orcish": "orkovskiy",
+		"scum": "svolotsch",
+		"scoundrel": "podlets",
+		"um": "vot",
+		"uh": "vot",
+		"faggot": "pidaraz'"
```

Comment:
remove this one. besides the fact that we shouldn't be mechanically acknowledging slurs, having these on the repo makes it valid for reports and takedowns.